axlearn/experiments/text/gpt/fuji.py

```diff
@@ -408,7 +408,7 @@ def get_trainer_kwargs(
             ),
             learner_kwargs=dict(peak_lr=3e-4, weight_decay=0.1),
             max_sequence_length=max_sequence_length,
-            train_batch_size=train_batch_size,
+            train_batch_size=32,
```

Comment:
temporary?

Reply:
Yes, just for small-scale testing, will remove before switching from draft state